gnomad_qc/v4/sample_qc/sex_inference.py

```diff
@@ -986,58 +872,33 @@ def main(args):
         type=int,
     )
 
-    sex_coverage_args = parser.add_argument_group(
-        "Sex imputation interval coverage",
-        "Arguments used for computing interval coverage for sex imputation.",
-    )
-    sex_coverage_args.add_argument(
-        "--sex-imputation-interval-coverage",
-        help=(
-            "Create a MatrixTable of interval-by-sample coverage on a specified list of contigs with PAR regions "
-            "excluded."
-        ),
-        action="store_true",
-    )
-    sex_coverage_args.add_argument(
-        "--normalization-contig",
-        help="Which autosomal chromosome to use for normalizing the coverage of chromosomes X and Y.",
-        type=str,
-        default="chr20",
-    )
-    sex_coverage_args.add_argument(
-        "--calling-interval-name",
-        help=(
-            "Name of calling intervals to use for interval coverage. One of: 'ukb', 'broad', or 'intersection'. Only "
-            "used if '--test' is set."
-        ),
-        type=str,
-        choices=["ukb", "broad", "intersection"],
-        default="intersection",
-    )
-    sex_coverage_args.add_argument(
-        "--calling-interval-padding",
-        help=(
-            "Number of base pair padding to use on the calling intervals. One of 0 or 50 bp. Only used if '--test' is "
-            "set."
-        ),
-        type=int,
-        choices=[0, 50],
-        default=50,
-    )
-
-    sex_interval_qc_args = parser.add_argument_group(
+    sex_imputation_interval_qc_args = parser.add_argument_group(
         "Sex chromosome interval QC",
         "Arguments used for making an interval QC HT from the sex imputation interval coverage MT.",
     )
-    sex_interval_qc_args.add_argument(
+    sex_imputation_interval_qc_args.add_argument(
         "--sex-imputation-interval-qc",
         help=(
             "Create a Table of the fraction of samples per interval and per platform with mean DP over thresholds "
             "specified by '--mean-dp-thresholds'."
         ),
         action="store_true",
     )
-    sex_interval_qc_args.add_argument(
+    sex_cov_mt_read_if_exists_action = sex_imputation_interval_qc_args.add_argument(
```

Comment:
I might just not know how to do it, I got an error when adding an argument with the same name 2 times. This was the way I found that would allow me to have the help message print in the way that I wanted it to